src/main/java/it/gov/pagopa/payhub/activities/connector/ionotification/mapper/NotificationRequestMapper.java

```diff
@@ -50,28 +56,49 @@ public List<NotificationRequestDTO> map(List<PaymentOptionDTO> paymentOptions, L
                 .toList();
     }
 
-    private static NotificationRequestDTO mapNotificationRequestDTO(
-            Long orgId, Long debtPositionTypeOrgId, IONotificationDTO ioNotificationDTO,
-            InstallmentDTO installmentDTO, NotificationRequestDTO.OperationTypeEnum operationType) {
+    private NotificationRequestDTO mapNotificationRequestDTO(DebtPositionDTO debtPositionDTO, IONotificationDTO ioNotificationDTO,
+                                                                    InstallmentDTO installmentDTO, NotificationRequestDTO.OperationTypeEnum operationType) {
 
         NotificationRequestDTO notificationRequestDTO = new NotificationRequestDTO();
         notificationRequestDTO.setFiscalCode(installmentDTO.getDebtor().getFiscalCode());
-        notificationRequestDTO.setOrgId(orgId);
-        notificationRequestDTO.setDebtPositionTypeOrgId(debtPositionTypeOrgId);
+        notificationRequestDTO.setOrgId(debtPositionDTO.getOrganizationId());
+        notificationRequestDTO.setDebtPositionTypeOrgId(debtPositionDTO.getDebtPositionTypeOrgId());
         if (ioNotificationDTO.getIoTemplateSubject() != null && ioNotificationDTO.getIoTemplateMessage() != null && ioNotificationDTO.getServiceId() != null) {
             notificationRequestDTO.setSubject(ioNotificationDTO.getIoTemplateSubject());
-            notificationRequestDTO.setMarkdown(ioNotificationDTO.getIoTemplateMessage());
+            notificationRequestDTO.setMarkdown(replacePlaceholders(ioNotificationDTO.getIoTemplateMessage(), installmentDTO, debtPositionDTO.getDescription()));
             notificationRequestDTO.setServiceId(ioNotificationDTO.getServiceId());
         }
         notificationRequestDTO.setAmount(installmentDTO.getAmountCents());
         notificationRequestDTO.setOperationType(operationType);
 
         return notificationRequestDTO;
     }
+
+    private String replacePlaceholders(String markdown, InstallmentDTO installment, String dpDescription) {
+        Map<String, String> placeholders = new HashMap<>();
+
+        placeholders.put("%posizioneDebitoria_descrizione%", Objects.toString(dpDescription, ""));
+        placeholders.put("%debitore_nomeCompleto%", Objects.toString(installment.getDebtor().getFullName(), ""));
+        placeholders.put("%debitore_codiceFiscale%", Objects.toString(installment.getDebtor().getFiscalCode(), ""));
+        placeholders.put("%importoTotale%", Objects.toString(centsAmountToEuroString(installment.getAmountCents()), ""));
+        placeholders.put("%IUV%", Objects.toString(installment.getIuv(), ""));
+        placeholders.put("%NAV%", Objects.toString(installment.getNav(), ""));
+        placeholders.put("%causale%", Objects.toString(installment.getRemittanceInformation(), ""));
+
+        String formattedDueDate = installment.getDueDate() != null ? installment.getDueDate().format(DateTimeFormatter.ofPattern("dd/MM/yyyy")) : "";
```

Comment:
let's move the placeHolder configuration on a separate class in order to immediate locate it in case of new placeHolder configuration.

Set the DateTimeFormtter as a class constant, it will be probably re-used (keep it public)